ssl-monitor/api/main.py

```
@@ -0,0 +1,350 @@
+            server_info=data.get("server"),
+            ip_info=json.dumps(data.get("ip_info", {})),
+            alerts=json.dumps(ssl_info.get("alerts", [])),
+            recommendations=json.dumps(data.get("recommendations", [])),
+            checked_at=datetime.fromisoformat(result.get("timestamp").replace("Z", "+00:00")),
+            error_message=result.get("error") if result.get("status") == "error" else None
+        )
+        db.add(db_check)
+        db.commit()
+        db.refresh(db_check)
+        return db_check.id
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)